pkgs/tools/virtualization/xe-guest-utilities/default.nix

```diff
@@ -6,16 +6,17 @@
 
 buildGoModule rec {
   pname = "xe-guest-utilities";
-  version = "7.30.0";
+  version = "8.3.1";
 
   src = fetchFromGitHub {
     owner = "xenserver";
     repo = "xe-guest-utilities";
     rev = "v${version}";
-    hash = "sha256-gMb8QIUg8t0SiTtUzqeh4XM5hHgCXuf5KlV3OeoU0LI=";
+    hash = "sha256-d0WdezcT44ExeHSnoJ3Dn0u/IRlhWreOZPSVw6Q1h/w=";
   };
 
-  vendorHash = "sha256-zhpDvo8iujE426/gxJY+Pqfv99vLNKHqyMQbbXIKodY=";
+  deleteVendor = true;
+  vendorHash = "sha256-X/BI+ZhoqCGCmJfccyEBVgZc70aRTp3rL5j+rBWG5fE=";
```

Comment:
Did you try setting vendorHash to null and did not work? Then we just keep it as is. If that turns out to work, we use that.

Reply:
I tried, but it gives out this error:
`build/gobuild/xenstore/xenstore.go:11:2: cannot find module providing package golang.org/x/sys/unix: import lookup disabled by -mod=vendor`

Reply:
Then we keep it